server/index.js

```diff
@@ -1,31 +1,54 @@
-const express = require("express");
-const cors = require("cors");
-const bodyParser = require("body-parser");
+const startTime = Date.now();
 
-const routes = require("./routes");
+const path = require("path");
+const express = require("express");
 
-const app = express();
+const app = require("./app");
+const database = require("./database");
 
 // utils
+const logger = require("./utils/logger");
 const logchimpConfig = require("./utils/logchimpConfig");
 const config = logchimpConfig();
-if (!config) {
-	console.log(
-		"LogChimp configuration missing!\nTry running this command 'logchimp install' again."
+
+// Serve vue app
+if (process.env.NODE_ENV === "production") {
+	app.use(express.static(path.resolve(__dirname, "public")));
+	app.get(/.*/, (req, res) =>
+		res.sendFile(path.resolve(__dirname, "public/index.html"))
 	);
-	process.exit(1);
 }
 
-// Set the default environment to be `development`
-process.env.NODE_ENV = process.env.NODE_ENV || "development";
-
-// contains key-value pairs of data submitted in the request body
-app.use(bodyParser.json());
-
-// enable all CORS requests
-app.use(cors());
-
-// import all routes
-app.use(routes);
-
-module.exports = app;
+database.migrate
+	.latest()
+	.then(() => {
+		logger.info("Database migration complete");
+	})
+	.catch(err => {
```

Comment:
Codacy has a fix for the issue: [Replace `err` with `(err)`](https://app.codacy.com/gh/logchimp/logchimp/pullRequest?prid=7771594)
```suggestion
	.catch((err) => {
```